docs/platform/howto/create_new_service.rst

```diff
@@ -0,0 +1,48 @@
+﻿Creating a new Aiven service
+============================
+
+You can add services for the various Aiven products in the Aiven web console.
+
+To create a new service:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/tools/console/console_create_service.png
+
+3. Select the main properties for your service:
+
+   a. Select the service type.
+
+      If there are several versions available, select the version that you want to use. By default, the latest available version is selected.
+
+   b. Select the cloud provider and region that you want to run your service on.
+
+      **Note:** The pricing for the same service may vary between different providers and regions. The service summary on the right side of the console shows you the pricing for your selected options.
+   c. Select a service plan.
+
+      This defines the number of servers and what kind of memory, CPU, and disk resources are allocated to your service.
+
+   d. Enter a name for your service.
+
+      A random name is provided by default, but you can enter a more recognizable name to distinguish it from other services.
+
+
+4. Click **Create Service** under the summary on the right side of the console.
+
+   This brings you back to the **Services** view. Your new service is listed with a status indicator to show that it is being created.
+
+5. Click the service name.
+
+   The *Overview* page for the service opens.
+
+   .. image:: /images/tools/console/console_service.png
+
+   This view shows you the connection parameters for your service, its current status, and the configuration options.
+
+   The status is *REBUILDING* while the service is being created for you. Once the service is ready, the status changes to *RUNNING*. While services typically start up in a couple of minutes, the performance varies between cloud providers and regions, and it may take longer in some circumstances.
```

Comment:
Is the upper-case, italic formatting indicating something in particular? I'm not sure I've seen this before and it looks a little out of place.

Reply:
The capitalization is just based on how it appears in the UI - it might not look ideal in the text, but it's better to reference UI texts as they are in the UI, even though the capitalization there isn't very uniform...

Reply:
I can't find anywhere it's capitalised when I use the console. Is there any danger this changed recently?
![services-status2](https://user-images.githubusercontent.com/172607/129366454-b1737375-a7ed-44a3-92e7-963cfdc7826f.png)
![services-status1](https://user-images.githubusercontent.com/172607/129366456-d7469816-b3e0-4dc6-a60a-35334f472a11.png)

Reply:
Hmm... Maybe that's changed since I created the initial version of the content - that's now updated ;)